include/deal.II/grid/grid_generator.h

```diff
@@ -96,13 +96,14 @@ namespace GridGenerator
              const bool                    colorize = false);
 
   /**
-   * \brief %Triangulation of a d-simplex with (d+1) vertices and mesh cells.
+   * \brief Create a d-simplex (a triangle in 2d, a tetrahedron in 3d) made
+   * out of (d+1) vertices and (d+1) cells.
```

Comment:
It is confusing to me that we write `(d + 1) vertices`: the mesh has more vertices than that. Could we just omit this and write that it is made out of `d + 1` cells?

Reply:
(I know what the author is getting at, it just took me a few seconds to figure it out)